esmvaltool/recipes/hydrology/recipe_marrmot.yml

```
@@ -45,24 +42,4 @@ diagnostics:
     scripts:
       script:
         script: hydrology/marrmot.py
-        basin: Meuse
-
-  diagnostic_hourly:
-    description: marrmot input preprocessor for hourly data
-    additional_datasets:
-      - {dataset: ERA5, project: OBS6, tier: 3, type: reanaly, version: 1}
-    variables:
-      tas: &var_hourly
-        mip: E1hr
-        start_year: 1990
-        end_year: 2018
-        preprocessor: hourly
-      pr: *var_hourly
-      # MARRMoT requires PET. Thus, we need psl, rsds, rsdt
-      psl: *var_hourly
-      rsds: *var_hourly
-      rsdt: *var_hourly
-    scripts:
-      script:
-        script: hydrology/marrmot.py
-        basin: Meuse
+        basin: meuse
```

```suggestion
        basin: Meuse
```
Previous comment may also affect this one.